README.md

```diff
@@ -21,7 +24,7 @@ To connect to an Amazon S3 compatible cloud storage service, you will need to sp
 | :---         |     :---     |
 | endpoint   | URL to object storage service.   | 
 | accessKeyID | Access key is the user ID that uniquely identifies your account. |   
```

Comment:
`secretAccessKey` was kept to be closer to how AWS S3 calls it. but it is simpler to call it the way you changed it. We can change this across all our libraries as well.